citar.el

```diff
@@ -222,6 +222,23 @@ If nil, single resources will open without prompting."
   :group 'citar
   :type '(boolean))
 
+(defcustom citar-open-note-functions
+  '(citar-file--open-note citar--open-note-field citar-org-roam-open-note)
+  "List of functions to open a note."
+  :group 'citar
+  :type '(function))
```

Comment:
Note we already have a defcustom below for this, which takes a single function. It's used now, for example, by org-roam users.